### ISSUE TITLE
[BUG/ISSUE] Properly resize toolbar

### ISSUE THREAD

**Issue by martinet101** [martinet101/WingetUI#301]
Properly resize toolbar

**Comment by ppvnf** [comment]
You mean something like this https://github.com/martinet101/WingetUI/issues/23#issuecomment-1266949626?

**Comment by martinet101** [comment]
No, i mean like hiding the labels when there's not enough room to show them

**Comment by martinet101** [comment]
Something similar like what happens not in the title, that shrinks to be smaller when the window is resized

**Comment by ppvnf** [comment]
I get it now:

![Captura de tela 2022-10-19 153532](https://user-images.githubusercontent.com/73800734/196775836-6e0e0fdf-6706-4c3f-8c0e-7df57c9b7a6e.png)

**Comment by martinet101** [comment]
But if you run the latest head it should look like this. The widgets automatically resize
![image](https://user-images.githubusercontent.com/53119851/196893370-144afb15-0de7-46c4-a499-9448c1f694b5.png)

**Comment by ppvnf** [comment]
> But if you run the latest head it should look like this. The widgets automatically resize

That's indeed fixed in the latest beta. 

The only small problems I saw were these:

Search bar not resizing properly at certain window sizes:
![Captura de tela 2022-10-20 052417](https://user-images.githubusercontent.com/73800734/196897161-094f1967-ac89-48b5-91ea-ef1fc81db6c1.png)

Toolbar itens like "Reset blacklist" could be shown in the situation below:
![Captura de tela 2022-10-20 052340](https://user-images.githubusercontent.com/73800734/196897441-6bd5a201-6c16-4207-b8dc-621c78f76d25.png)

**Comment by martinet101** [comment]
Done!

> Toolbar itens like "Reset blacklist" could be shown in the situation below:

Selective label showing can't be set in Qt's Toolbar



> Search bar not resizing properly at certain window sizes:

The search bar is expected to shrink, but i have increates the minimum width from 15px to 100 because it wa too small
